notebooks/cleaning_data.ipynb

```diff
@@ -0,0 +1,2892 @@
+{
```

Comment:
<p>You just filter the indirect missing data to only include numeric features instead of updating it. Update the comments, please.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mawada-sweis/Clustering-Analysis/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>